sim/paladin/righteous_vengeance.go

```diff
@@ -0,0 +1,75 @@
+package paladin
+
+import (
+	"time"
+
+	"github.com/wowsims/sod/sim/core"
+	"github.com/wowsims/sod/sim/core/proto"
+)
+
+const RVTicks = 4
+
+func (paladin *Paladin) registerRV() {
+	if !paladin.hasRune(proto.PaladinRune_RuneCloakRighteousVengeance) {
+		return
+	}
+
+	paladin.RegisterAura(core.Aura{
+		Label:    "Righteous Vengeance",
+		Duration: core.NeverExpires,
+		OnReset: func(aura *core.Aura, sim *core.Simulation) {
+			aura.Activate(sim)
+		},
+		OnSpellHitDealt: func(aura *core.Aura, sim *core.Simulation, spell *core.Spell, result *core.SpellResult) {
+
+			if spell.Flags.Matches(SpellFlag_RV) && result.DidCrit() {
+				paladin.procRV(sim, result)
+			}
+		},
+	})
+
+	paladin.rv = paladin.RegisterSpell(core.SpellConfig{
+		ActionID:    core.ActionID{SpellID: 440675},
+		SpellSchool: core.SpellSchoolHoly,
+		DefenseType: core.DefenseTypeMelee,
+		ProcMask:    core.ProcMaskSpellDamage,
+		Flags:       core.SpellFlagPureDot | core.SpellFlagIgnoreAttackerModifiers,
+
+		// SpellFlagIgnoreTargetModifiers was thought to be used based on wowhead flags
+		// WCL parses show that this is not the case
+
+		DamageMultiplier: 1,
+		ThreatMultiplier: 1,
+
+		Cast: core.CastConfig{
+			IgnoreHaste: true,
+		},
```

Comment:
This shouldm't be needed since it doesn't have a cast time

Reply:
Ignite has it, I think it has to do with the Dot Not ticking faster with haste

Reply:
Oh oops. AFAIK all dots don't tick faster with haste in vanilla. The only thing that field affects is cast time
https://github.com/wowsims/sod/blob/a0e8093b28521bfd54d65ffffabc7fa0ac117162/sim/core/cast.go#L162-L166

So it's probably just a leftover from removing that stuff but no worries!